gapis/replay/interfaces.go

```diff
@@ -33,7 +33,7 @@ type Support interface {
 
 	// CanReplayOn returns true if the API can be replayed on the specified
 	// device.
```

Comment:
Please document what the `MemoryLayout` represents. I assume it is of the device that took the capture. If so, would it make more sense to pass that?